### PR TITLE
fix(config): update pod utilities version to latest

### PR DESCRIPTION
**Pull request by zuc**
Signed-off-by: Michele Zuccala <michele@zuccala.com>

It seems that @poiana forgot to bump some images here (#630) and this causes most Prow jobs to fail.
This PR fixes the issue.

image

/cc @FedeDP
/assign @leogr

### CODE REVIEW

**Comment by FedeDP** [comment]
Note that this morning i opened an issue regarding this: https://github.com/falcosecurity/test-infra/issues/634

**Comment by zuc** [comment]
Also, /cc @maxgio92

**Comment by zuc** [comment]
This is probably gonna need a force-merge since it fixes the reason why tests are failing

**Comment by zuc** [comment]
> Note that this morning i opened an issue regarding this: #634

@FedeDP if you look at other autobump-authored PRs in the past, poiana also bumped `config/config.yaml` but in #630 it didn't happen. This probably requires a little bit more investigation.

**Comment by maxgio92** [comment]
[This

**Comment by maxgio92** [comment]
> This is probably gonna need a force-merge since it fixes the reason why tests are failing.

I agree

**Review by leogr — APPROVED** [review]
/approve

**Comment by leogr** [comment]
Note for other maintainers:

The check-prow-config job is not currently working, and this PR is needed to fix the problem. Since @poiana cannot merge the PR with the usual flow (because the required job is filing), I will force the merging to overcome the issue. Hoping this will work :crossed_fingers: 

/override check-prow-config

**Pull request merged**
